PrestaShop/PrestaShop
# All shops checkboxes for combination edit/bulk-edit forms

| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Integrate "Apply to all shops" checkboxes for combination edit/bulk-edit forms. Detailed some remaining issues in "how to test section" bellow the table. ![Screenshot from 2022-12-16 12-59-14](https://user-images.githubusercontent.com/31609858/208084055-64c8e595-0767-433c-b7b8-1ff330501ab3.png)
| Type?             | refacto
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | 
| Related PRs       | 
| How to test?      | Check bellow.
| Possible impacts? | 

**How to test**
Bulk combination form and edit combination form should now have checkboxes when clicked on fields that are multishop. Checkboxes saying "apply to all shops" which when checked, will apply the value to all the shops that combination is associated to. There are some **issues** that should be solved in dedicated PR:
1. `Disabling switch doesn't trigger the "all shops checkbox" to appear`, but it should. This part should be handled in dedicated PR. (for example when you want to disable "low stock threshold" for all combinations you cannot do it now because of this issue. Same goes for bulk action, if you will just enable the field and not click on related input, the checkbox won't appear). ping @jolelievre this is the bug we've discussed during one daily meetup. :arrow_up: 
2. `The combinations list is visually broken` since this PR https://github.com/PrestaShop/PrestaShop/pull/27719. It needs some tweaking when adding multishop checkboxes to the list in a dedicated PR. https://github.com/PrestaShop/PrestaShop/pull/27719 looks like this (its not related to my changes):
![Screenshot from 2022-12-16 17-32-46](https://user-images.githubusercontent.com/31609858/208133392-5bde365d-87f9-4bed-beb7-46cebedba341.png)
